### PR TITLE
feature(pluign-flow-builder): handoff payloads

### PR DESCRIPTION
**Pull request by Iru89**
## Description
Be able to use a payload action defined in flow-builder in onFinishPayload of handoff. 
If there is a linked content it will have priority over the one defined as action payload.

## Context
With action payload I mean the text defined in "action after agent assistance"
Captura de Pantalla 2023-05-18 a las 10 15 43

## Testing

The pull request...

- has unit tests
- has integration tests
- doesn't need tests because... **[provide a description]**

### CODE REVIEW

File: packages/botonic-plugin-flow-builder/src/index.ts [hunk 1]
```diff
@@ -129,6 +125,18 @@ export default class BotonicPluginFlowBuilder implements Plugin {
...
+    if (content && 'buttons' in content) {
+      content.buttons.forEach(async button => {
+        if (button.payload) {
+          const contentButton = await this.getContent(button.payload)
+          if (contentButton?.type === NodeType.PAYLOAD) {
+            button.payload = contentButton.content.payload
+          }
+        }
+      })
+    }
```
Comment (vanbasten17) [inline packages/botonic-plugin-flow-builder/src/index.ts:138]: extract this into a function 🙏

**Review by vanbasten17 — APPROVED** [review]
Thanks @Iru89 for the improvement and detecting it! Let's keep it up 💪

File: packages/botonic-plugin-flow-builder/src/flow-builder-models.ts [hunk 2]
```diff
@@ -126,6 +126,7 @@ export interface HandoffNodeContent {
   message: TextLocale[]
   fail_message: TextLocale[]
```
Comment (AinaVendrell) [inline packages/botonic-plugin-flow-builder/src/flow-builder-models.ts:127]: this no longer exists on the handoff content. We should remove it in a future PR

**Review by AinaVendrell — APPROVED** [review]

File: packages/botonic-plugin-flow-builder/src/flow-builder-models.ts (hunk 2 above)
Comment (Iru89) [reply packages/botonic-plugin-flow-builder/src/flow-builder-models.ts:127]: ok

**Pull request merged**